roles/installer/molecule/satellite/prepare.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Prepare
+  hosts: all
+  gather_facts: true
+  become: true
+  vars:
+    foreman_repositories_version: '2.4'
+  roles:
+    - theforeman.operations.puppet_repositories
+    - theforeman.operations.foreman_repositories
+  post_tasks:
+    - name: Add satellite-installer alias
+      copy:
+        dest: /bin/satellite-installer
```

Comment:
this could also be just a symlink, right?

also, the "real" package has it in `/usr/sbin/satellite-installer`

and do we want to also mimick `/usr/sbin/capsule-certs-generate`?